src/main/kotlin/no/nav/syfo/application/SendDialogmeldingArenaCronjob.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.syfo.application
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.cronjob.Cronjob
+import no.nav.syfo.application.cronjob.CronjobResult
+import no.nav.syfo.db.DatabaseInterface
+import no.nav.syfo.model.ReceivedDialogmelding
+import no.nav.syfo.persistering.db.getUnpublishedArenaMeldinger
+import no.nav.syfo.persistering.db.lagreSendtArena
+import no.nav.syfo.services.ArenaDialogmeldingService
+import no.nav.syfo.util.safeUnmarshal
+import org.slf4j.LoggerFactory
+
+class SendDialogmeldingArenaCronjob(
+    private val database: DatabaseInterface,
+    private val arenaDialogmeldingService: ArenaDialogmeldingService,
+) : Cronjob {
+    override val initialDelayMinutes: Long = 2
+    override val intervalDelayMinutes: Long = 10
+
+    override suspend fun run() {
+        val result = runJob()
+        log.info(
+            "Completed sending dialogmelding to arena with result: {}, {}",
+            StructuredArguments.keyValue("failed", result.failed),
+            StructuredArguments.keyValue("updated", result.updated),
+        )
+    }
+
+    suspend fun runJob(): CronjobResult {
+        val result = CronjobResult()
+        val unpublishedArenaMeldinger = database.getUnpublishedArenaMeldinger()
+        try {
```

Comment:
Usikker på om det har noe å si, men burde try/catch ligge inn i .forEach her? Vi vil i hvert fall at én feilende melding ikke skal feile hele jobben...

Reply:
Ja, godt poeng! Flikka litt mye frem og tilbake her, det gir kanskje mer mening å ha den inni. Kan skrive test også 👍🏼

Reply:
Ja, synes det er best med try-catch inni for-løkka for slike cronjob'er.

Reply:
Bra sett, egentlig! Det ble en fin test ut av det også 🙌🏼